installation/scripts/minikube.ps1

```diff
@@ -45,15 +45,10 @@ function StartMinikube() {
         + " --memory ${MEMORY}"`
         + " --cpus 4"`
         + " --extra-config=apiserver.Authorization.Mode=RBAC"`
-        + " --extra-config=apiserver.GenericServerRunOptions.CorsAllowedOriginList='.*'"`
-        + " --extra-config=controller-manager.ClusterSigningCertFile='/var/lib/localkube/certs/ca.crt'"`
-        + " --extra-config=controller-manager.ClusterSigningKeyFile='/var/lib/localkube/certs/ca.key'"`
-        + " --extra-config=apiserver.admission-control='LimitRanger,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota'"`

```

Comment:
Why those extra configs are removed?

Reply:
i explained it in the commit comments:
cors: new syntax is required (--extra-config "apiserver.cors-allowed-origins=http://*")

ClusterSigningCert: not required since minikube 28.x (e.g. mentioned here: https://github.com/istio/istio.io/pull/2708#issue-219216989)

Admission-control: the settings we used are now default for the api server